scripts/upload-zynq7000-smt2.gdb

```diff
@@ -0,0 +1,20 @@
+#openocd gdb server with onboard zynq7000 Zedboard Digilent FTDI-based SMT2 JTAG
+target remote localhost:3333
+set verbose on
+
+set output-radix 16
+set pagination on
+
+#Reset USB phy
+set *((u32*)0xE000A040) = 0
+
+monitor adapter srst pulse_width 250
+monitor adapter srst delay 250
+
+monitor reset
+monitor halt
```

Comment:
I'd use single command
`monitor reset halt`

Reply:
I would leave that, because on error we will se exact point where monitor failed.